snap/kubelet.yaml

```diff
@@ -35,125 +35,115 @@ parts:
+      add-arg bool enable-load-reader
+      add-arg string enforce-node-allocatable
+      add-arg string event-burst
+      add-arg string event-qps
+      add-arg string event-storage-age-limit
+      add-arg string event-storage-event-limit
+      add-arg string eviction-hard
+      add-arg string eviction-max-pod-grace-period
+      add-arg string eviction-minimum-reclaim
+      add-arg string eviction-pressure-transition-period
+      add-arg string eviction-soft
+      add-arg string eviction-soft-grace-period
+      add-arg bool fail-swap-on
+      add-arg string feature-gates
+      add-arg string file-check-frequency
+      add-arg string global-housekeeping-interval
+      add-arg string google-json-key
+      add-arg string hairpin-mode
+      add-arg string healthz-bind-address
+      add-arg string healthz-port
+      add-arg string host-ipc-sources
+      add-arg string host-network-sources
+      add-arg string host-pid-sources
+      add-arg string http-check-frequency
+      add-arg string image-gc-high-threshold
+      add-arg string image-gc-low-threshold
+      add-arg string init-config-dir
+      add-arg string iptables-drop-bit
+      add-arg string iptables-masquerade-bit
+      add-arg string kube-api-burst
+      add-arg string kube-api-content-type
+      add-arg string kube-api-qps
+      add-arg string kube-reserved
+      add-arg string kube-reserved-cgroup
+      add-arg string kubelet-cgroups
+      add-arg bool log-cadvisor-usage
+      add-arg string machine-id-file
+      add-arg bool make-iptables-util-chains
+      add-arg string manifest-url
+      add-arg string manifest-url-header
+      add-arg string max-open-files
+      add-arg string max-pods
+      add-arg string minimum-image-ttl-duration
+      add-arg string node-status-update-frequency
+      add-arg string oom-score-adj
+      add-arg string pod-cidr
+      add-arg string pod-manifest-path
+      add-arg string pods-per-core
+      add-arg string port
+      add-arg bool protect-kernel-defaults
+      add-arg string read-only-port
+      add-arg string registry-burst
+      add-arg string registry-qps
+      add-arg string resolv-conf
+      add-arg string rkt-api-endpoint
+      add-arg string rkt-path
+      add-arg string runtime-request-timeout
+      add-arg bool serialize-image-pulls
+      add-arg string storage-driver-buffer-duration
+      add-arg string storage-driver-db
+      add-arg string storage-driver-host
+      add-arg string storage-driver-password
+      add-arg bool storage-driver-secure
+      add-arg string storage-driver-table
+      add-arg string storage-driver-user
+      add-arg string streaming-connection-idle-timeout
+      add-arg string sync-frequency
+      add-arg string system-cgroups
+      add-arg string system-reserved
+      add-arg string system-reserved-cgroup
+      add-arg string tls-cert-file
+      add-arg string tls-private-key-file
+      add-arg string volume-stats-agg-period
```

Comment:
This was just a find/replace of `ensure_arg` -> `add-arg`. None of the args in here have changed.